04_removeFromArray/removeFromArray.js

```diff
@@ -1,6 +1,16 @@
-const removeFromArray = function() {
-
+const removeFromArray = function (...values) {
+  /* This line declares function that can accept multiple arguments and puts them in an array */
+  const array =
```

Comment:
`.shift()` modified original array (values). And modifying arguments is usually a bad practice. Can you find a way to get the first argument and the rest of the arguments in the function declaration?